### DISCUSSION TITLE
Does it supports Apple Silicon Mac?

### DISCUSSION THREAD

**Discussion by mylife4aiur5** [mattyamonaca/layerdivider#13] [General]
So far, I dont see anything on my WebUI so I have to wonder if it only works with Windows or not.

**Comment by itemx** [comment]
I haven't tried but the installation/launch script in PowerShell basically installs from the requirements.
So, maybe, possible when the dependencies are ready on ARM macOS?

The pytoshop on Windows requires Visual Studio SDK, not certain of the dependency on macOS pip.
